src/style/_colors.scss

```diff
@@ -165,6 +165,8 @@ $color-primary--900: #003a45;
   PLANNING_700: $color-planning--700;
   PLANNING_800: $color-planning--800;
   PLANNING_900: $color-planning--900;
+
+  GRID: $color-grid;
```

Comment:
This variable was missing, which is why the charts had no horizontal grid lines showing.